test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
Ahem … `test_change_rpath` is the method these diff lines belong to and that's where I've been missing this. (And that's also why this isn't hidden yet in the PR view.) :wink: